src/components/PrintButton/PrintButton.js

```diff
@@ -23,26 +23,38 @@ class PrintButton extends React.Component {
     onBeforeGetContent: noop,
   };
 
+  eventObject = {};
+
   getContent = () => {
     return this.props.contentRef.current;
   };
 
+  handlePrintBeforeGetContent = () => {
+    this.eventObject.event.stopPropagation();
```

Comment:
Doesn't this break the functionality in other places where the button is used?

Reply:
This is done so that the click is not propagated to table row (which is also clickable). It currently works. 
What's your suggestion?

Reply:
Print button is used in different places. And I just want to be sure that your changes do not brake functionality of this button in other places.
Could you confirm that it does not brake anything?

Reply:
Tested. It works at existing places.